cmd/functional-test/testcases.txt

```diff
@@ -0,0 +1,49 @@
+www.example.com {{binary}} -silent 
+www.example.com {{binary}} -silent -l test-data/request.txt
+www.example.com {{binary}} -silent -request test-data/raw-request.txt
+www.example.com {{binary}} -silent -response-time
+www.example.com {{binary}} -silent -title
+www.example.com {{binary}} -silent -location
+www.example.com {{binary}} -silent -cname
+www.example.com {{binary}} -silent -cdn
+www.example.com {{binary}} -silent -probe
+www.example.com {{binary}} -silent -no-fallback
+www.example.com {{binary}} -silent -match-code 200, -mc 200
+www.example.com {{binary}} -silent -match-length 100, -ml 100
+www.example.com {{binary}} -silent -match-string xyz, -ms xyz
+www.example.com {{binary}} -silent -match-regex bbb, -mr bbb
+www.example.com {{binary}} -silent -filter-code 401, -fc 401
+www.example.com {{binary}} -silent -filter-length 23, -fl 23
```

Comment:
I think the comma shouldn't be necessary here

Reply:
Done.